rad/src/operators/string.rs

```diff
@@ -127,7 +130,27 @@ fn json_to_cbor(value: &json::JsonValue) -> Value {
         }
         json::JsonValue::Short(value) => Value::Text(String::from(value.as_str())),
         json::JsonValue::String(value) => Value::Text(String::from(value.as_str())),
-        json::JsonValue::Number(value) => Value::Float((*value).into()),
+        json::JsonValue::Number(value) => {
+            // This trick allows to tell whether a number has decimals by checking whether the
+            // mantissa contains more significant bits than the exponent can express.
```

Comment:
I thought you may be reimplementing `value.as_i64()`, but then I realized we are using `json`, not `serde_json`. No idea why.

https://docs.serde.rs/serde_json/struct.Number.html#method.as_i64